### PR TITLE
add weaviate tutorial

### PR DESCRIPTION
**Pull request by samos123**

### CODE REVIEW

File: docs/tutorials/weaviate.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+      engine: OLlama
+      resourceProfile: cpu:2
+```
+
+Note: It's important that you name the models as `text-embedding-ada-002` and `gpt-3.5-turbo` as Weaviate expects these names.
+
+Run the following command to deploy KubeAI:
+```bash
+helm upgrade --install kubeai kubeai/kubeai \
+    -f ./kubeai-values.yaml --reuse-values
```
Comment (nstogner) [inline docs/tutorials/weaviate.md:54]: No need for `--reuse-values` here.

File: docs/tutorials/weaviate.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
+# Weaviate with local autoscaling embedding and generative models
+
+Weaviate is a vector search engine that can integrate seamlessly with KubeAI's embedding and generative models. This tutorial demonstrates how to deploy both KubeAI and Weaviate in a Kubernetes cluster, using KubeAI as the OpenAI endpoint for Weaviate.
```
Comment (nstogner) [inline docs/tutorials/weaviate.md:3]: Maybe add a quick blurb about why you might want to do this: all of your data stays within your cluster.

**Review by nstogner — COMMENTED** [review]
LGTM, 2 comments, otherwise feel free to merge

File: docs/tutorials/weaviate.md (hunk 1 above)
Comment (samos123) [reply docs/tutorials/weaviate.md:54]: Hmm I kept it so people don't accidentally delete their existing models.

**Pull request merged**